wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/WireRun.kt

```diff
@@ -242,6 +248,7 @@ data class WireRun(
         .prune(treeShakingRubbish)
         .since(since)
         .until(until)
+        .only(onlyVersion)
```

Comment:
Similarly, it feels like all three of these need the version suffix.

Reply:
I think `pruningRules` can keep the 3 of them without suffix, but WireRun would have them all 3 with the suffix. Sounds good to you?

Reply:
Works for me

Reply:
fixed